### ISSUE TITLE
Typo in the project wiki's example OpenOCD command.

### ISSUE THREAD

**Issue by WRansohoff** [kendryte/openocd-kendryte#9]
This project's wiki looks like it has a small typo in the "Debugging with GDB" section. I think that it recommends using an OpenOCD command which does not seem to work with the current `master` branch, but I could be wrong about the source of the issue:

https://github.com/kendryte/openocd-kendryte/wiki/Debugging-with-GDB

Instead of `openocd -f kendryte.cfg -m0`, it looks like the command should be `openocd -f kendryte.cfg -m 0`. Without the space between `-m` and `0`, I get the following error when trying to connect:

```
Kendryte Open On-Chip Debugger For RISC-V v0.2.3 (2019-02-21)
Licensed under GNU GPL v2
adapter speed: 3000 kHz
Info : [...J-Link info...]
Info : VTarget = 3.356 V
Info : clock speed 3000 kHz
Info : JTAG tap: riscv.cpu tap/device found: 0x04e4796b (mfg: 0x4b5 (<unknown>), part: 0x4e47, ver: 0x0)
Error: Debug RAM write failed. Hardware error?
Warn : Failed to read misa at 0x301; trying 0xf10.
Error: Debug RAM write failed. Hardware error?
Error: Failed to read misa at 0xf10.
```

I could be wrong about why the OpenOCD command is failing, but using `-m 0` instead of `-m0` to select the core seems to fix things.

Steps to reproduce, in case I'm doing something else wrong:

1. Connect Ground, TDI, TDO, TCK, TMS, and RST pins to J-Link.
2. Power the board using its USB-C connector and connect JLink's voltage reference pin.
3. Attempt to connect with, `openocd -f kendryte.cfg -m0`.

**Comment by minux** [comment]
Fixed.

Thanks for reporting!

**Comment by WRansohoff** [comment]
Apologies, I think that I might have mis-reported this issue. I built the project from source and observed some inconsistencies, but I'm no longer convinced that the errors I saw were caused by passing in `-m0` instead of `-m 0`.

When I reverted commit ID 08bc16c3057be19927d6d5784e31abc208242719, I was able to connect to the chip successfully using either command.

Sorry for the confusion, but I think it is possible that this change may have introduced a regression:

https://github.com/kendryte/openocd-kendryte/commit/08bc16c3057be19927d6d5784e31abc208242719

I observed this behavior on an `aarch64` host platform, if that matters.
